packages/foreman/rubygem-dynflow/rubygem-dynflow.spec

```diff
@@ -5,7 +5,7 @@
 
 Summary: DYNamic workFLOW engine
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.8.36
+Version: 0.8.37
 Release: 2%{?foremandist}%{?dist}
```

Comment:
Didn't notice the release which should have been set to 1 before releasing :(

Reply:
Oh, neither did I. Will we try to fix it somehow or will we let it slide?

Reply:
Doesn't hurt and we should try to fix it at the next release.

Reply:
Ok, adding it to my checklist so I won't forget it